src/ucp/rma/flush.c

```diff
@@ -167,7 +161,6 @@ static unsigned ucp_ep_flush_resume_slow_path_callback(void *arg)
 {
     ucp_request_t *req = arg;
 
-    ucp_ep_flush_slow_path_remove(req);
```

Comment:
what if ucp_flush_check_completion() returns 0, seems like progress will not be removed?

Reply:
yes, you're right
should we use `UCS_CALLBACKQ_FLAG_ONESHOT` flag?

Reply:
maybe, pls check that it will keep current behavior

Reply:
ok